app/common/renderer/actions/SessionInspector.js

```diff
@@ -327,9 +328,35 @@ export function applyClientMethod(params) {
       window.dispatchEvent(new Event('resize'));
       return commandRes;
     } catch (error) {
-      log.error(error);
-      showError(error, {methodName: params.methodName, secs: 10});
-      dispatch({type: METHOD_CALL_DONE});
+      if (!getState().inspector.autoSessionRestart) {
```

Comment:
I assume here needs to detect what kind of error should try out this idea.

For example, when you do open a url command for a session which doesn't support the endpoint. Then the session gets an error. Such kind of command error should not get restarted

Reply:
Thanks for the suggestion! I’ve added a condition to handle these cases now, checking for:
- error.message === "Cannot read properties of undefined (reading 'apply')"
- error.includes "process is not running (probably crashed)"